authentication/src/main/AndroidManifest.xml

```diff
@@ -17,4 +17,39 @@
 
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
+    <uses-permission android:name="android.permission.INTERNET" />
+
+    <queries>
+        <package android:name="com.ubercab" />
```

Comment:
Are we doing fingerprint validation against these before talking to them? We should be. If that's already in place somewhere else, please point me to it, if not, can you have me be an explicit reviewer on that?

Reply:
We are invoking the Uber apps via applinks. If the Uber app is not registered for that applink (via assetlinks.json) it will not be invoked so fingerprint validation is not required.

Reply:
* verified for that applink